docs/plugins/woodpecker-plugins/plugins.json

```diff
@@ -35,6 +35,11 @@
       "docs": "https://codeberg.org/qwerty287/woodpecker-regex-check/raw/branch/main/docs.md",
       "verified": false
     },
+    {
+      "name": "Gitea comment",
+      "docs": "https://github.com/markopolo123/gitea-comment-plugin/blob/main/README.md",
```

Comment:
we need a "docs.md" at least the metadata head and it would be nice to have an example too ...

just look what you get when you look at the other url's

Reply:
updated repo with docs and example.